examples/eight_schools/eight_schools.py

```diff
@@ -0,0 +1,94 @@
+"""implement the stan 8 schools example
```

Comment:
Can you rephrase as: `Implement the 8 schools example using the recommended non-centered parameterization.:.`